src/main/webapp/i18n/de/plagiarism.json

```diff
@@ -46,6 +46,10 @@
             "fetchingResults": "Plagiate werden geladen...",
             "replyToAccusation": "Stellung zu dem Vorwurf nehmen",
             "replyToAccusationDueDateNotification": "Du kannst dich spätestens bis zum <b>{{ dueDate }}</b> zu dem Vorwurf äußern.",
+            "continuousPlagiarismControl": "Kontinuierliche Plagiatskontrolle",
+            "continuousPlagiarismControlTooltip": "Die kontinuierliche Plagiatskontrolle löst jede Nacht automatisch eine Plagiatskontrolle aus. Detected are marked as plagiarism and students are gently notified about this fact.",
```

Comment:
Is it intended that the translation is German + English mixed up?

Reply:
Done